po/fur.po

```diff
@@ -0,0 +1,868 @@
+msgid "Failed to connect to Avahi server: %s"
+msgstr "Impussibil conetisi al servidôr Avahi: %s"
+
+#: avahi-ui/avahi-ui.c:735
+msgid "Browsing for services on <b>local network</b>:"
+msgstr "Esplorazion dai servizis su <b>rêt locâl</b>:"
+
+#: avahi-ui/avahi-ui.c:737
+#, c-format
+msgid "Browsing for services in domain <b>%s</b>:"
+msgstr "Esplorazions dai servizis tal domini <b>%s</b>:"
+
+#: avahi-ui/avahi-ui.c:773
+#, c-format
+msgid "Failed to create browser for %s: %s"
+msgstr "Impussibil creâ l'esploradôr par %s: %s"
+
+#: avahi-ui/avahi-ui.c:907
+#, c-format
+msgid "Failed to create resolver for %s of type %s in domain %s: %s"
+msgstr "Impussibil creâ il risolvidôr par %s di gjenar %s tal domini %s: %s"
+
+#: avahi-ui/avahi-ui.c:982
+#, c-format
+msgid "Failed to create domain browser: %s"
+msgstr "Impussibil creâ l'esploradôr di dominis: %s"
+
+#: avahi-ui/avahi-ui.c:993
+msgid "Change domain"
+msgstr "Cambie domini"
+
+#: avahi-ui/avahi-ui.c:1043 avahi-ui/avahi-ui.c:1180
+msgid "Browsing..."
+msgstr "Esplorazion..."
+
+#: avahi-ui/avahi-ui.c:1047 avahi-ui/bssh.c:149 avahi-ui/bssh.c:156
+#: avahi-ui/bssh.c:161
+msgid "_Cancel"
+msgstr "_Anulâ"
+
+#: avahi-ui/avahi-ui.c:1048
+msgid "_OK"
+msgstr "_Va ben"
+
+#: avahi-ui/avahi-ui.c:1134
+msgid "Initializing..."
+msgstr "Inizializazion..."
+
+#: avahi-ui/avahi-ui.c:1162
+msgid "Location"
+msgstr "Posizion"
+
+#: avahi-ui/avahi-ui.c:1167 avahi-utils/avahi-browse.c:553
+msgid "Name"
+msgstr "Non"
+
+#: avahi-ui/avahi-ui.c:1172 avahi-utils/avahi-browse.c:553
+msgid "Type"
+msgstr "Gjenar"
+
+#: avahi-ui/avahi-ui.c:1184
+msgid "_Domain..."
+msgstr "_Domini..."
+
+#: avahi-ui/bssh.c:55
+#, c-format
+msgid ""
+"%s [options]\n"
+"\n"
+"    -h --help            Show this help\n"
+"    -s --ssh             Browse SSH servers\n"
+"    -v --vnc             Browse VNC servers\n"
+"    -S --shell           Browse both SSH and VNC\n"
+"    -d --domain=DOMAIN   The domain to browse in\n"
+msgstr ""
+"%s [opzions]\n"
+"\n"
+"    -h --help            Mostre chest jutori\n"
+"    -s --ssh             Esplore servidôrs SSH\n"
+"    -v --vnc             Esplore servidôrs VNC\n"
+"    -S --shell           Esplore sedi SSH che VNC\n"
+"    -d --domain=DOMINI   Il domini dulà lâ a esplorâ\n"
+
+#: avahi-ui/bssh.c:101 avahi-utils/avahi-browse.c:775
+#, c-format
+msgid "Too many arguments\n"
+msgstr "Masse argoments\n"
+
+#: avahi-ui/bssh.c:149
+msgid "Choose Shell Server"
+msgstr "Sielzi il servidôr shell"
+
+#: avahi-ui/bssh.c:149 avahi-ui/bssh.c:156 avahi-ui/bssh.c:161
+msgid "C_onnect"
+msgstr "C_oneti"
+
+#: avahi-ui/bssh.c:151
+msgid "Desktop"
+msgstr "Scritori"
+
+#: avahi-ui/bssh.c:152
+msgid "Terminal"
+msgstr "Terminâl"
+
+#: avahi-ui/bssh.c:156
+msgid "Choose VNC server"
+msgstr "Sielzi il servidôr VNC"
+
+#: avahi-ui/bssh.c:161
+msgid "Choose SSH server"
+msgstr "Sielzi il servidôr SSH"
+
+#: avahi-ui/bssh.c:185
+#, c-format
+msgid "Connecting to '%s' ...\n"
+msgstr "Conession a \"%s\" ...\n"
```

Comment:
@tmtfx just out of curiosity I wonder why the single quotes are replaced with the escaped double quote?

(It doesn't matter much and I've already merged the PR but I'm just curious. Does Weblate suggest that automatically or something like that?)

Reply:
nope, in Italy and therefore also in Friuli, people use double quotation marks. At most « and » are used, but single quotation marks are not used, although they are intelligible.